weedcoco/index/indexing.py

```diff
@@ -0,0 +1,140 @@
+import argparse
+import os
+import pathlib
+import json
+import requests
+from math import ceil
+
+
+class ElasticSearchIndex:
```

Comment:
I had imagined what is now in weedcoco/index to live with the webapp stuff... It's about the webapp, not about the format or repository... but I suppose it can live together? It's a bit confusing, because the celery stuff in django will need to call this.

Reply:
Yeah that's the goal. I'll make adjustments if the current structure can't be called by celery and django appropriately.